README.md

```diff
@@ -1 +1,134 @@
-# project-catwalk
\ No newline at end of file
+# project-catwalk
+# Contributers
+[Christian Peterson](https://github.com/ChrisRPeterson), [Amanda Davitt](https://github.com/acdavitt), [Dylan Reid](https://github.com/dylanreid7), [Julia Thea Boyadjan](https://github.com/julia-thea)
+
+# About
+The objective of Project Catwalk is to build a new retail website for a fictitous clothing company, J.CAD.  A team of four software engineers is given a time frame of three weeks to complete the request and adhere to the specifications laid out in the business requirement document.  The website is comprised of four main widgets.  Further details about each widget can be found below:
```

Comment:
I just felt the beginning of this paragraph was a tad confusing since it used present tense and "the business requirement document" wouldn't mean anything to a reader unfamiliar with HR or the assignment.  I'm imagining that the intention of the README is for a reader to understand it after the project is completed and if they have no context of the assignment or HR.
Suggestion: "The objective of Project Catwalk was to build a new retail website for a fictitious clothing company, J.CAD.  Our team of four software engineers was given a time frame of three weeks to complete the request.  We adhered to the specifications laid out in a business requirements document assigned by our school, Hack Reactor."